src/openforms/authentication/tests/test_utils.py

```diff
@@ -71,3 +80,23 @@ def test_get_cosign_info(self):
                 kwargs={"form_slug": "form-with-cosign"},
             ),
         )
+
+    def test_remove_auth_info_from_session(self):
+        factory = APIRequestFactory()
+        request = factory.get("/foo")
+        request.session = self.client.session
+        request = Request(request)
```

Comment:
```suggestion
```
`APIRequestFactory` already produces a `Request` instance, no?

I'd expect your type checker to complain about this too btw

Reply:
I thought so too, that is why I used `APIRequestFactory` instead of `RequestFactory` but strangely enough my type checker complains about this if I don't explicitly cast it:

![Screenshot from 2024-05-23 14-23-20](https://github.com/open-formulieren/open-forms/assets/29249171/cb2c94e6-c539-4f98-bdaa-13a7564900c0)

Reply:
I'll add `AnyRequest` :+1:

Reply:
So this means my type checker isn't messed up? :smile: